ui/src/app/store/user/effects.ts

```diff
@@ -87,7 +86,7 @@ export class UserListEffect {
             this.authService.logOut();
             return [];
           }
-          return [deleteUserSuccess({ userId }), loadApplications(), loadUsersEntityAction()];
```

Comment:
as I remember loadApplications is fired to get correct roles. please check roles works correctly if you delete owner of organization; admin of org; user of org; owner of the app, etc. You can view in git history for which bug this line was added

Reply:
Fixed

Reply:
what is the difference between `loadApplications` and `loadApplicationsNotLoader`?

Reply:
They differ from each other in that loadApplications() runs the preloader and loadApplicationsNotLoader() doesn't.

Reply:
are you talking about setting `isPending: true`, in reducer by loadApplications? is this the only difference?

Reply:
Yes.

Reply:
sounds like a 'plaster fix'. lets zoom and discuss

Reply:
Ok.